config/vufind/MyResearchMenu.ini

```diff
@@ -0,0 +1,71 @@
+; Optional configuration for user account menu. If there are no items, or this file is even not used. VuFind uses
```

Comment:
I think two sentences might need to be joined together here. Should it read:

"If there are no items defined, or if this file is missing, VuFind will use default settings..."

Reply:
Definitely. The first sentence is a leftover from time before I wrote the documentation.